components/dashboard/src/data/configurations/configuration-queries.ts

```diff
@@ -60,6 +60,23 @@ export const useConfiguration = (configurationId: string) => {
     });
 };
 
+export const useDeleteConfiguration = (configurationId: string) => {
```

Comment:
I think the better pattern here is to pass arguments into the mutationFn vs. the top level hook. It's what we're doing in our other delete mutations. See [here](https://github.com/gitpod-io/gitpod/blob/67168240c69b3726e056ecb863229bc8bc7c1a78/components/dashboard/src/data/auth-providers/delete-org-auth-provider-mutation.ts#L15-L43) for an example.